slider.js

```diff
@@ -30,115 +46,97 @@ const createSlider = (function () {
-    image.alt = "Placeholder image";
-    imageContainer.appendChild(image);
-    imageParent.appendChild(imageContainer);
-
-    sliderRoot.querySelector(SLIDER_CLASSES.NAVIGATION_BUTTONS).remove();
+  const createSlide = (src, alt) => {
+    const slide = document.createElement("li");
+    slide.classList.add(".slider-item");
+    slide.innerHTML = `<img
+        width="400"
+        height="200"
+        class="image"
+        src="${src}"
+        alt="${alt}"
+      />`;
+
+    return slide;
   };
 
-  const getPrevSlideIndex = (activeSlideIndex, slides) => {
+  const getPrevSlideIndex = () => {
     return activeSlideIndex === 0 ? slides.length - 1 : activeSlideIndex - 1;
   };
 
-  const getNextSlideIndex = (activeSlideIndex, slides) => {
+  const getNextSlideIndex = () => {
+    console.log();
     return activeSlideIndex >= slides.length - 1 ? 0 : activeSlideIndex + 1;
   };
 
-  const selectSlide = (slides, newSlideIndex) => {
+  const selectSlide = (newSlideIndex) => {
     slides.forEach((slide) =>
       slide.classList.remove(SLIDER_CLASSES.ITEM_ACTIVE)
     );
     slides[newSlideIndex].classList.add(SLIDER_CLASSES.ITEM_ACTIVE);
+    activeSlideIndex = newSlideIndex;
   };
 
-  const getPrevSlide = (slides, activeSlideIndex) => {
-    const prevSlideIndex = getPrevSlideIndex(activeSlideIndex, slides);
-    selectSlide(slides, prevSlideIndex);
-
-    return prevSlideIndex;
-  };
-
-  const getNextSlide = (slides, activeSlideIndex) => {
-    const nextSlideIndex = getNextSlideIndex(activeSlideIndex, slides);
-    selectSlide(slides, nextSlideIndex);
-
-    return nextSlideIndex;
+  const selectPrevSlide = () => {
+    const prevSlideIndex = getPrevSlideIndex();
+    selectSlide(prevSlideIndex);
+    initAutoscroll();
   };
 
-  const autoScrollHandler = (slides, activeSlideIndex) => {
-    setInterval(() => {
-      activeSlideIndex = getNextSlide(slides, activeSlideIndex);
-    }, 3000);
+  const selectNextSlide = () => {
+    const nextSlideIndex = getNextSlideIndex();
+    selectSlide(nextSlideIndex);
+    initAutoscroll();
   };
 
-  const init = (sliderRoot, customConfig) => {
-    let activeSlideIndex = 0;
-
-    const defaultConfig = {
-      showNavigationButtons: false,
-      autoScroll: true,
-      customPlaceholder: "https://via.placeholder.com/400x200",
-    };
-
-    const config = Object.assign(defaultConfig, customConfig || {});
+  const initPlaceholder = () => {
+    if (slides.length > 0) {
+      return;
+    }
 
-    const slides = getSlides(sliderRoot);
-    const buttons = getButtons(sliderRoot);
+    const placeholderSlide = createSlide(config.placeholder, "Placeholder");
+    const list = sliderRoot.querySelector(SLIDER_CLASSES.LIST);
+    list.appendChild(placeholderSlide);
+    slides = getSlides();
+  };
 
-    if (slides.length === 0) {
-      getPlaceholder(sliderRoot, config.customPlaceholder);
+  const initAutoscroll = () => {
+    if (!config.autoScroll || slides.length === 0) {
```

Comment:
Should be: 
`if (!config.autoScroll || slides.length < 2)`

Reply:
right